### PR TITLE
Add Kimono Desktop Releases v1.0.5

### PR DESCRIPTION
**Pull request by tennisonchan**
#### Adding a new cask
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download Casks/kimono.rb` is error-free.
- [x] `brew cask style --fix Casks/kimono.rb` left no offenses.
- [x] `brew cask install Casks/kimono.rb` worked successfully.
- [x] `brew cask uninstall Casks/kimono.rb` worked successfully.

### CODE REVIEW

File: Casks/kimono.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'kimono' do
+  version '1.0.5'
+  sha256 '30c6b83661573e10e1afabad0ef7f5c9ead60c4b2cedd4454160736c6c91a55f'
+
+  url 'https://s3.amazonaws.com/kimono-desktop-releases/v1.0.5/Kimono.app.zip'
```
Comment (adidalal) [inline Casks/kimono.rb]: use version interpolation
also, needs URL comment

Comment (adidalal) [inline Casks/kimono.rb]: Needs a `# s3.amazonaws.com/kimono-desktop-releases was verified as official when first introduced to the cask` comment above this line ([example](https://github.com/caskroom/homebrew-cask/blob/4ad02c9f087deb9e553ab0129c079d73c474add8/Casks/0ad.rb#L5); [reasoning](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/url.md#when-url-and-homepage-hostnames-differ-add-a-comment)).

**Comment by tennisonchan** [comment]
Cannot run `brew cask style --fix kimono` due to the following error.
```shell
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/specification.rb:2007:in `raise_if_conflicts': Unable to activate rubocop-cask-0.5.0, because rubocop-0.39.0 conflicts with rubocop (~> 0.37.0) (Gem::LoadError)
  from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/specification.rb:1176:in `activate'
  from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems.rb:186:in `rescue in try_activate'
  from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems.rb:183:in `try_activate'
  from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:132:in `rescue in require'
  from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:144:in `require'
  from /Users/home/.gem/ruby/2.0.0/gems/rubocop-0.39.0/lib/rubocop/config_loader_resolver.rb:16:in `block in resolve_requires'
  from /Users/home/.gem/ruby/2.0.0/gems/rubocop-0.39.0/lib/rubocop/config_loader_resolver.rb:12:in `each'
  from /Users/home/.gem/ruby/2.0.0/gems/rubocop-0.39.0/lib/rubocop/config_loader_resolver.rb:12:in `resolve_requires'
  from /Users/home/.gem/ruby/2.0.0/gems/rubocop-0.39.0/lib/rubocop/config_loader.rb:39:in `load_file'
  from /Users/home/.gem/ruby/2.0.0/gems/rubocop-0.39.0/lib/rubocop/config_store.rb:24:in `options_config='
  from /Users/home/.gem/ruby/2.0.0/gems/rubocop-0.39.0/lib/rubocop/cli.rb:64:in `act_on_options'
  from /Users/home/.gem/ruby/2.0.0/gems/rubocop-0.39.0/lib/rubocop/cli.rb:25:in `run'
  from /Users/home/.gem/ruby/2.0.0/gems/rubocop-0.39.0/bin/rubocop:14:in `block in <top (required)>'
  from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/benchmark.rb:296:in `realtime'
  from /Users/home/.gem/ruby/2.0.0/gems/rubocop-0.39.0/bin/rubocop:13:in `<top (required)>'
  from /Users/home/.gem/ruby/2.0.0/bin/rubocop:23:in `load'
  from /Users/home/.gem/ruby/2.0.0/bin/rubocop:23:in `<main>'
Error: style check failed
```

**Comment by adidalal** [comment]
Pinging @jawshooah about the `rubocop` error

**Comment by tennisonchan** [comment]
Checked! Ran `brew cask style --fix kimono` on another computer.

screen shot 2016-04-22 at 9 31 24 am

**Comment by vitorgalvao** [comment]
You have `v1.0.5` in `url`, but the value in version `differs`. Why?

**Comment by tennisonchan** [comment]
@vitorgalvao Good call. Amended.

**Pull request merged**